app/cdap/components/GridTable/components/TransformationToolbar/index.tsx

```diff
@@ -37,11 +37,12 @@ export default function({
 }: ITransformationToolBarProps) {
   const classes = useStyles();
   const [showName, setShowName] = useState<boolean>(false);
-  const [anchorEl, setAnchorEl] = useState<HTMLElement>(null);
+  const [anchorEl, setAnchorEl] = useState<HTMLElement[]>(null);
   const [selectedMenuOptions, setSelectedMenuOptions] = useState<IMenuItem[]>([]);
 
   const toggleMenu = () => {
     setSelectedMenuOptions([]);
+    setAnchorEl(null);
```

Comment:
rename setAnchorEl to setAnchorElement completely across the green lines in the PRs

Reply:
Done